File: src/main/java/com/jump2digital/Hackathon22/models/Center.java
package com.jump2digital.Hackathon22.models;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import org.springframework.data.mongodb.core.mapping.Document;

@Document
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Entity
public class Center {

    @Id
    private Long centerId;
    private Integer cityCode;
    private Integer regionCode;
    private String centerType;
    private Integer opArea;
}

File: app/Models/Center.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Jenssegers\Mongodb\Eloquent\Model as Eloquent;

class Center extends Eloquent
{
    use HasFactory;
    protected $fillable = [
        'center_id',
        'city_code',
        'region_code',
        'center_type',
        'op_area',
    ];
    //protected $primaryKey = 'id';

}
